templates/email_new_request_is_received.html

```diff
@@ -0,0 +1,7 @@
+<p>Hi {{ user_name }},</p>
+<br>
+<p>You've received a new mentorship request!</p>
```

Comment:
you could also add a paragraph like a "call to action" for the user to go to the app and see the new request (not a link to the request, right now is not feasible, but just a message saying something like "See the new request on the mentorship application!")